### PR TITLE
feat: Added average ratings and no of ratings

### LINKED ISSUE #6523: Organizer Session Overview: Re-add rating columns to all session tables
The organizer session overview page does not have the column for ratings anymore. Please re-add rating columns to all session tables and ensure the feature works.

The feature was originally implemented here https://github.com/fossasia/open-event-frontend/pull/3198

Screenshot_2019-08-01 Confirmed Sessions AddOwnerEvent Events Open Event

### PR DESCRIPTION
**Pull request by Creatoon**
Fixes #6523 

#### Short description of what this resolves:
This pr adds no of ratings and average ratings column on the session page.

Screenshot_2021-01-30 Session Sessions brio Events Open Event

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Creatoon** [comment]
@iamareebjamal sir please review , thanks

**Comment by iamareebjamal** [comment]
Merge them in 1 component

File: app/controllers/events/view/sessions/list.js [hunk 1]
```diff
@@ -60,6 +60,18 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
...
+        width         : 90,
+        valuePath     : 'averageRating',
+        cellComponent : 'ui-table/cell/events/view/sessions/cell-rating-details'
+      },
+      {
+        name          : this.l10n.t('No. of ratings'),
+        width         : 90,
+        valuePath     : 'feedbacks.length',
+        cellComponent : 'ui-table/cell/events/view/sessions/cell-rating-details'
+      },
```
Comment (iamareebjamal) [inline app/controllers/events/view/sessions/list.js:74]: This should be one column

Comment (Creatoon) [reply app/controllers/events/view/sessions/list.js:74]: @iamareebjamal sir by one column, do you mean like this,

Screenshot_2021-01-30 Session Sessions brio Events Open Event(1)

Comment (iamareebjamal) [reply app/controllers/events/view/sessions/list.js:74]: Rating and Average Rating/No. of rating should all be in one column

Comment (Creatoon) [reply app/controllers/events/view/sessions/list.js]: ok, i am making the changes

**Comment by Creatoon** [comment]
@iamareebjamal sir after adding them in one column
Screenshot_2021-01-30 Session Sessions brio Events Open Event(2)

**Comment by afzal442** [comment]
@Creatoon sort by is missing.

**Comment by Creatoon** [comment]
> @iamareebjamal sir after adding them in one column
> Screenshot_2021-01-30 Session Sessions brio Events Open Event(2)

@iamareebjamal Sir in one column, they are looking very messed up, i think.

**Comment by Creatoon** [comment]
> @Creatoon sort by is missing.

Sort by works according to value path and in one column the value path is ID and what's the need of sorting the ratings according to id's

**Comment by afzal442** [comment]
No that's not. I told you according to the feature mentioned by Sir. Maybe I was wrong there. Sorry for that.

**Comment by Creatoon** [comment]
> No that's not. I told you according to the feature mentioned by Sir. Maybe I was wrong there. Sorry for that.

not possible if i add them in one column

**Comment by iamareebjamal** [comment]
Make valuePath average rating

**Comment by Creatoon** [comment]
> Make valuePath average rating

Sir addRating function will throw error if i do this because addRating function takes valuePath as an  argument session_id and due to this addRating function will not able to recognize the id and unable to find the session to update its ratings.

**Comment by Creatoon** [comment]
@iamareebjamal sir you can see that , because of incorrect id(averageRating from valuePath), the session is not detected.
Screenshot from 2021-01-31 02-48-27

**Comment by Creatoon** [comment]
@iamareebjamal sir you can see that , because of incorrect id(averageRating from valuePath), the session is not detected.

Screenshot from 2021-01-31 02-48-27

**Comment by Creatoon** [comment]
@iamareebjamal sir sorting by average ratings is working.

Screenshot_2021-02-03 Session Sessions saasda asdasda Events Open Event

**Comment by iamareebjamal** [comment]
Just show `Rating` in header

Show the number of rating below the stars. Use minimum space as possible

**Comment by Creatoon** [comment]
@iamareebjamal  Sir, is it fine now?
Screenshot_2021-02-03 Session Sessions saasda asdasda Events Open Event(1)

**Comment by iamareebjamal** [comment]
Center it vertically as well

**Comment by Creatoon** [comment]
@iamareebjamal Sir vertically aligned now.
Screenshot_2021-02-04 Session Sessions Mr robot Events Open Event(1)

**Comment by Creatoon** [comment]
@iamareebjamal sir ?

**Comment by iamareebjamal** [comment]
What about admin section sessions page?

**Comment by Creatoon** [comment]
@iamareebjamal sir i have added ratings for admin page also , but on rating the stars, it shows unexpected errors , sometimes stars gets in two rows and also this error.

Screenshot from 2021-02-05 13-30-33

**Comment by maze-runnar** [comment]
it's not the problem with admin section. this problem rises when same user tries to rate session in organizer session section, and then goes to admin session section and try to rate same section there too. Then there when update rating should be called it is calling add rating.

**Comment by maze-runnar** [comment]
https://github.com/fossasia/open-event-frontend/pull/7350

**Pull request closed without merging**